dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -86,6 +86,14 @@ class DALIGenericIterator(object):
                  change during execution. If True, the mxnet.ndarray will be resized accordingly
                  if the shape of DALI returned tensors changes during execution.
                  If False, the iterator will fail in case of change.
+    last_batch_padded : bool, optional, default = False
+                 Whether the last batch in DALI is padded by some dummy values
+                 or it just wraps up. It tells when next epoch should start from
```

Comment:
`It tells when ...`
I find the whole description quite confusing.

Reply:
Improved a bit. Check now.